apps/authentication-portal/src/main/webapp/identifierauth.jsp

```diff
@@ -47,36 +47,43 @@
 <script>
     function submitIdentifier () {
         var isEmailUsernameEnabled = JSON.parse("<%= isEmailUsernameEnabled %>");
+        var isSaaSApp = JSON.parse(getParameterByName("isSaaSApp").toLowerCase());
         var tenantName = getParameterByName("tenantDomain");
+
         var userName = document.getElementById("username");
         var usernameUserInput = document.getElementById("usernameUserInput");
-        var isSaaSApp = JSON.parse(getParameterByName("isSaaSApp").toLowerCase());
 
         if (usernameUserInput) {
             var usernameUserInputValue = usernameUserInput.value.trim();
 
-            if (!isSaaSApp && tenantName) {
+            if (tenantName && tenantName !== "null" && tenantName !== undefined) {
```

Comment:
Do we need all three checks here?

Reply:
"tenantName !== undefined" not needed. I forgot to remove it. Fixed it.